aiopg/sa/result.py

```diff
@@ -5,9 +5,18 @@
 
 from sqlalchemy.sql import expression, sqltypes
 
+from ..utils import PY_35
 from . import exc
 
 
+try:
+    StopAsyncIteration
```

Comment:
Why do you need the check?

`if PY_35:` statement above prevents `StopAsyncIteration` usage for python < 3.5